package.json

```diff
@@ -4,11 +4,17 @@
   "description": "The JavaScript Implementation of IPLD",
   "leadMaintainer": "Volker Mische <volker.mische@gmail.com>",
   "main": "src/index.js",
+  "types": "./dist/src/index.d.ts",
   "pre-push": [
     "lint",
     "test"
   ],
+  "files": [
```

Comment:
I remember @rvagg saying that he prefers not using `files` and include tests as well. Is there a reason to use `files`?

Reply:
See - https://github.com/ipld/js-ipld-raw/pull/50#discussion_r592022924

In this case it turns out there's an `.npmignore` file in this repo already so files isn't necessary.  I can remove it if required, or we can remove `.npmignore` as it duplicates a lot of what's in `.gitignore`.

Reply:
I see. `files` is OK then with me.